rust/lance-encoding/src/compression_algo/fsst/src/fsst.rs

```diff
@@ -968,6 +984,10 @@ fn decompress_bulk(
         if in_curr < in_end {
             // last code cannot be an escape code
             let code = compressed_strs[in_curr] as usize;
+            let required_space = *out_curr + lens[code] as usize;
+            if required_space > out.len() {
+                out.resize(required_space.max(out.len() * 2), 0);
+            }
```

Comment:
Do we still need this?  When would we not have enough size?

Reply:
removed